Formula/nginx-unit.rb

```diff
@@ -0,0 +1,267 @@
+class NginxUnit < Formula
+  desc "Dynamic web and application server for multiple languages"
+  homepage "https://unit.nginx.org/"
+  url "https://unit.nginx.org/download/unit-1.1.tar.gz"
+  sha256 "7c66365f5ea87e7e8903bac9d639676947fe2ab7aa799ce35defc426b3409ee0"
+
+  depends_on "openssl"
+  depends_on "pcre"
+
+  def install
+    openssl = Formula["openssl"]
+    pcre = Formula["pcre"]
+
+    system "./configure",
+      "--prefix=#{prefix}",
+      "--bindir=#{bin}",
+      "--sbindir=#{bin}",
+      # This tells the unit daemon where it can load modules from. Configuring
+      # the location at runtime is painful. We want this to be in the
+      # $HOMEBREW_PREFIX so that other formulae can link in additional modules. 
+      "--modules=#{HOMEBREW_PREFIX/"lib/unit"}",
+      "--state=#{var/"unit"}",
+      "--log=#{var/"log/unit.log"}",
+      "--pid=#{var/"run/unit.pid"}",
+      "--control=unix:#{var/"run/unit.control.sock"}",
+      "--cc-opt=-I#{pcre.opt_include} -I#{openssl.opt_include}",
+      "--ld-opt=-L#{pcre.opt_lib} -L#{openssl.opt_lib}"
+
+    system "make", "install"
+
+    # Beyond here we want to install modules into the keg, so we'll move them
+    # into the prefix ourselves
+    libunit = lib/"unit"
+    libunit.mkpath
+
+    # There is no system Go
+    #system "./configure", "go"
+    #system "make", "go"
+    #libunit.install "build/go.unit.so"
+
+    system "./configure", "perl"
+    system "make", "perl"
+    libunit.install "build/perl.unit.so"
+
+    # We can't build against the system PHP, it doesn't have the right SAPI
```

Comment:
Remove these lines.

Reply:
It's worth leaving a note, it might seem like the php module should be installed if macOS has php, but it's in the caveats, too.